src/server_manager/web_app/app.ts

```diff
@@ -1024,6 +1037,9 @@ export class App {
       this.appRoot.showNotification(this.appRoot.localize('saved'));
       serverView.accessKeyDataLimit = dataLimitToDisplayDataAmount(limit);
       this.refreshTransferStats(this.selectedServer, serverView);
+      // Don't display the feature collection disclaimer anymore.
+      serverView.shouldShowFeatureMetricsDisclaimer = false;
+      window.localStorage.setItem('dataLimits-feature-collection-disclaimer', 'true');
```

Comment:
Let's rename `-disclaimer` to `-consent`. Consent is really what we want to track.

Reply:
Done.